ccd-definition/ComplexTypes/CareSupervision/OtherDocument.json

```diff
@@ -12,7 +12,7 @@
     "LiveFrom": "01/01/2017",
     "ID": "OtherDocument",
     "ListElementCode": "dateTimeUploaded",
-    "FieldType": "Text",
+    "FieldType": "DateTime",
```

Comment:
@tomaszpowroznik is this going to cause issues with the importing of definitions with elastic search?

Reply:
AFAIK yes, but we can just ask to destroy and rebuild index. Can be tested on local, happy to check